ADOPTERS.md

```diff
@@ -6,6 +6,8 @@ A non-exhaustive unordered list of organizations that have adopted OpenTelemetry
 
 [EcoBee](https://www.ecobee.com/) [uses the OpenTelemetry Java SDK](https://www.honeycomb.io/blog/bees-working-together-how-ecobees-engineers-adopted-honeycomb/), and the collector both as a centralized service and as a sidecar.
 
+[F5](https://www.f5.com/)
```

Comment:
please provide details about which components are being used (collector, specific language SDKs)

Reply:
Absolutely, @yurishkuro! I just updated the PR with the components that are currently implemented and being used.